### DISCUSSION TITLE
My account has been flagged

### DISCUSSION THREAD

**Discussion by souraavsen** [community/community#123829] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My account is flagged. My account was hacked before but now its been recovered. I need my account back from being marked as flagged because my work kinda depends on this. I can not do any Git-related actions. And I saw that most people get responses in more than 2 months unfortunately so I am scared I won't get it as soon as I need it. This is ridiculous.

Anyone know any action I can take so they respond quickly? All the things are depended on my account. But its more than 2 months its been flagged and unacceptable but there are no reply from the github support. My profile is not accessable.

Thanks in advance.

**Comment by souraavsen** [comment]
@queenofcorgis

**Reply by ebndev** [reply]
Hi @souraavsen,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding! 

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.
